lib/uwb/UwbSession.cxx

```diff
@@ -32,11 +32,7 @@ UwbSession::AddPeer(UwbMacAddress peerMacAddress)
 {
     std::scoped_lock peersLock{ m_peerGate };
     PLOG_VERBOSE << "adding peer with address " << peerMacAddress.ToString();
```

Comment:
Since you're here, can you modify the peer-related plog prints in this file to include the session id in a consistent message?
```suggestion
    PLOG_VERBOSE << "Session with id " << m_sessionId << " added peer with address " << peerMacAddress.ToString();
```